src/icalendar/tests/test_encoding.py

```diff
@@ -7,30 +7,32 @@
 import os
 import pytz
 
+@pytest.mark.parametrize('field, expected_value', [
+    ('PRODID', '-//Plönë.org//NONSGML plone.app.event//EN'),
+    ('X-WR-CALDESC', 'test non ascii: äöü ÄÖÜ €'),
+])
+def test_calendar_from_ical_respects_unicode(field, expected_value, calendars):
+    cal = calendars.calendar_with_unicode
+    assert cal[field].to_ical().decode('utf-8') == expected_value
+
+@pytest.mark.parametrize('test_input, field, expected_value', [
+    ('event_with_unicode_fields', 'SUMMARY', 'Non-ASCII Test: ÄÖÜ äöü €'),
+    ('event_with_unicode_fields', 'DESCRIPTION', 'icalendar should be able to handle non-ascii: €äüöÄÜÖ.'),
+    ('event_with_unicode_fields', 'LOCATION', 'Tribstrül'),
+    # Non-unicode characters in summary
+    # https://github.com/collective/icalendar/issues/64
+    ('issue_64_event_with_non_unicode_summary', 'SUMMARY', 'åäö'),
+    # Unicode characters in summary
+    ('issue_64_event_with_unicode_summary', 'SUMMARY', 'abcdef'),
```

Comment:
I noticed that `src/icalendar/tests/events/issue_64_event_with_non_unicode_summary.ics` contains (as the name suggest) not unicode characters but this tests for `'åäö'` and then the unicode test only checks ascii characters. This should be the other way around? Why are the tests not failing?

Reply:
I think by `unicode` I've meant `non-ascii`, so I'll fix that.

Reply:
The test where actually wrong. `issue_64_event_with_non_ascii_summary.ics` was called `issue_64_event_with_unicode_summary.ics` and contained
```
BEGIN:VEVENT
SUMMARY:åäö
END:VEVENT
```

and originally the test was `('issue_64_event_with_unicode_summary', 'SUMMARY', 'abcdef'),`, where summary clearly didn't match. I looked into the action log to figure out, why the action wasn't failing. After pulling the code myself to debug it, I noticed that they where indeed not equal, but you forgot to assert the equality.

Can you please add the assert for this test.